### PR TITLE
refactor: use string templates for logging messages

### PR DESCRIPTION
**Pull request by ursjoss**
A house-keeping PR: use string templates instead of string concatenation for logging.

Expected to have a minor positive effect on memory consumption.

Note: I applied Netbeans refactorings on Warnings in Netbeans.

### CODE REVIEW

File: modules/au.com.trgtd.tr.email/src/au/com/trgtd/tr/email/task/FetchEmailThread.java [hunk 1]
```diff
@@ -43,12 +43,8 @@ public FetchEmailThread() {
      */
     @Override
     public void run() {
-//      LOG.info("Starting email fetch at: " + new Date());
```
Comment (RadekCap) [inline modules/au.com.trgtd.tr.email/src/au/com/trgtd/tr/email/task/FetchEmailThread.java:46]: Why this deletions?

File: modules/au.com.trgtd.tr.email/src/au/com/trgtd/tr/email/utils/RenderableMessage.java [hunk 2]
```diff
@@ -59,18 +59,18 @@ private boolean extractPart(final Part part) throws MessagingException, IOExcept
 
         if (part.getContent() instanceof Multipart) {
 
-//            LOG.info("EXTRACT PART (" + n + ") " + "Multipart");
+//            LOG.log(Level.INFO, "EXTRACT PART ({0}) Multipart", n);
```
Comment (RadekCap) [inline modules/au.com.trgtd.tr.email/src/au/com/trgtd/tr/email/utils/RenderableMessage.java:62]: Ugh.

I would either delete or uncomment those single-liners of logging. Maybe in a standalone change and everywhere so we can easily find it in history later when needed.

**Review by RadekCap — CHANGES_REQUESTED** [review]
There are just small questions/suggestions before merging. Thank you.

File: modules/au.com.trgtd.tr.email/src/au/com/trgtd/tr/email/task/FetchEmailThread.java (hunk 1 above)
Comment (ursjoss) [reply modules/au.com.trgtd.tr.email/src/au/com/trgtd/tr/email/task/FetchEmailThread.java:46]: @RadekCap Your two comments indicate the struggle I was going though myself. My first approach was to actually delete the logs but then decided to migrate them in a best shot appraoch. I would like to remove the commented out code, but in the context of a different PR. I did not revert the initial deletion in this file but in the others...

File: modules/au.com.trgtd.tr.email/src/au/com/trgtd/tr/email/utils/RenderableMessage.java (hunk 2 above)
Comment (ursjoss) [reply modules/au.com.trgtd.tr.email/src/au/com/trgtd/tr/email/utils/RenderableMessage.java:62]: Exactly

File: modules/au.com.trgtd.tr.email/src/au/com/trgtd/tr/email/task/FetchEmailThread.java (hunk 1 above)
Comment (ursjoss) [reply modules/au.com.trgtd.tr.email/src/au/com/trgtd/tr/email/task/FetchEmailThread.java:46]: I restored it for now

File: modules/au.com.trgtd.tr.email/src/au/com/trgtd/tr/email/utils/RenderableMessage.java (hunk 2 above)
Comment (ursjoss) [reply modules/au.com.trgtd.tr.email/src/au/com/trgtd/tr/email/utils/RenderableMessage.java:62]: Do you agree to still migrate them here and delete the migrated ones later in a different PR?

Comment (RadekCap) [reply modules/au.com.trgtd.tr.email/src/au/com/trgtd/tr/email/utils/RenderableMessage.java:62]: yes, that's a good idea to have them separated.

**Review by RadekCap — APPROVED** [review]
Good to go, thank you.

**Comment by ursjoss** [comment]
Thanks for the review @RadekCap!

**Pull request merged**